templates/expo-template-default/components/Drawer.tsx

```diff
@@ -0,0 +1,54 @@
+import { Ionicons } from "@expo/vector-icons";
+import { PropsWithChildren, useState } from "react";
+import {
+  LayoutAnimation,
```

Comment:
use reanimated

Reply:
we should avoid using LayoutAnimation yeah